info/buildSrc/src/main/kotlin/org/cqfn/diktat/generation/docs/GenerationDocs.kt

```diff
@@ -220,3 +222,5 @@ private fun findBoldOrItalicText(regex: Regex,
     }
     return correctedLine
 }
+
+private fun String.getFirstNumber() = replace("\\s+".toRegex(), "").takeWhile { it.isDigit() || it == '.' }
```

Comment:
```suggestion
private fun String.getFirstNumber() = trimStart().takeWhile { it.isDigit() || it == '.' }
```
Isn't `trim` enough here? If I understand correctly, you extract rule number, which should be close to the start of the string.